style(*): grammar, spelling, punctuation (#539)
discordjs/guide: guide/commando/args.md

```js
super(client, {
	name: 'say',
	aliases: ['parrot', 'copy'],
	group: 'first',
	memberName: 'say',
	description: 'Replies with the text you provide.',
	args: [
		{
			key: 'text',
			prompt: 'What text would you like the bot to say?',
			type: 'string',
		},
	],
});
```

See? Simple.

- `key` is the name of the argument. When you define it in your `run` method, this is what you'll be using.

> - `prompt` is the text that displays if the user provides no argument. If someone uses `?say` by itself, that prompt will ask for the text.

Reviewer: ```suggestion
- `prompt` is the text that displays if the user doesn't provide arguments. If someone uses `?say` by itself, that prompt will ask for the text.
```

- `key` is the name of the argument. When you define it in your `run` method, this is what you'll be using.
- `prompt` is the text that displays if no argument is provided. If someone uses just `?say`, that prompt will come up asking for the text.

> - `type` is the type the argument is a part of. This can be many things, including `string`, `integer`, `user`, 'member`, etc.

Reviewer 2: ```suggestion
- `type` is the type the argument is a part of. This can be many things, including `string`, `integer`, `user`, `member`, etc.
```
Revert this change and others on the page.

Adding more args is as simple as adding another object to the array, like so:

<!-- eslint-skip -->

> "`js

Reviewer 2: ```suggestion
```js
```

<!-- eslint-skip -->

> "`js

Reviewer 2: ```suggestion
```js
```
